composer.json

```diff
@@ -26,7 +26,7 @@
         "zendframework/zend-mail": "^2.6.1",
         "zendframework/zend-validator": "^2.6",
         "friendsofphp/php-cs-fixer": "~1.7.0",
-        "phpunit/PHPUnit": "~4.0",
+        "phpunit/phpunit": "^4.6 || ^5.5",
```

Comment:
Ahh sorry I miss this comment. Well then, we can change all the `getMockBuilder` methods as `createMock`.